extensions/wikia/FandomCreator/Hooks.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace FandomCreator;
+
+use Wikia\DependencyInjection\Injector;
+use Wikia\Service\Gateway\UrlProvider;
+use WikiaDispatchableObject;
+
+class Hooks {
+	const SERVICE_NAME = "content-graph-service";
+
+	public static function onNavigationApiGetData(WikiaDispatchableObject $dispatchable) {
+		self::api();
+	}
+
+	private static function getEntityPath($entityId) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "getEntityPath" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1144)